test/verifiedTests/analysis/testfindMIIS/testfindMIIS.m

```diff
@@ -0,0 +1,53 @@
+% The COBRAToolbox: testfindMIIS.m
+%
+% Purpose:
+%     - testfindMIIS tests the debugging tool findMIIS that takes as input
+%     an infeasible model and returns to  Minimal Irreducible Infeasible
+%     Submodel
+%
+% Author:
+%     - Marouen BEN GUEBILA 02/12/2017
+
+global CBTDIR
+global ILOG_CPLEX_PATH
+
+addpath(genpath(ILOG_CPLEX_PATH));
```

Comment:
This is not required, and shouldn't be done. The path is added and set in `changeCobraSolver`.